### PR TITLE
Adding nightly circleCI job to run integration-mainnet tests at 12pm …

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**
…everyday

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -86,3 +86,24 @@ workflows:
...
             - init
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 12 * *"
+          filters:
+            branches:
+              only:
+                - develop
+    jobs:
```
Comment (atulsriv) [inline .circleci/config.yml:97]: can we put the integration test also?

Comment (tinystarinagalaxy) [reply .circleci/config.yml:97]: example job runs the Testnet and mainnet, no?

Comment (atulsriv) [reply .circleci/config.yml:97]: sorry, i thought this was a different repo entirely 😕

**Review by aspin — COMMENTED** [review]
could you separate testnet/mainnet as different jobs as well while we're at it? so we can require that only 1 of the 2 jobs succeed for PR builds

**Review by aspin — COMMENTED** [review]
I think we should really separate the nightly changes from the proto ones. The nightly changes can be merged in nowish (though I'd like the mainnet/testnet) separated as different jobs, and the proto changes need more discussion

**Pull request closed without merging**